randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -605,15 +621,25 @@ Extra - asset_id: collision_camera_014
           Enabled Remove Surface Crumble Blocks
           Any of the following:
               Single-wall Wall Jump (Beginner) or Wall Jump (Beginner) or Climb Rooms Vertically (No High Jump)
-              High Jump Boots and Super Jump (Beginner)
-              # Freeze https://youtu.be/8ehWaf87lqM
+              All of the following:
+                  # HJ combinations
+                  High Jump Boots
+                  Any of the following:
+                      Super Jump (Intermediate)
+                      Morph Ball and Unmorph Extend (Beginner)
+              # Freeze Moheek: https://youtu.be/8ehWaf87lqM
               Stand on Frozen Enemy (Intermediate) and Fully Freeze Enemy
   > Tunnel to Cavern Alcove
       All of the following:
           Morph Ball
           Any of the following:
               Single-wall Wall Jump (Beginner) or Wall Jump (Beginner) or Climb Rooms Vertically (No High Jump)
-              High Jump Boots and Super Jump (Beginner)
+              All of the following:
+                  # HJ combinations
+                  High Jump Boots
+                  Any of the following:
+                      Super Jump (Intermediate)
+                      Morph Ball and Underwater Bomb Jump (Intermediate)
```

Comment:
Wrong trick. Should be IBJ.

Reply:
Wrong. this should've been Unmorph Extend.